### ISSUE TITLE
Remove deprecated v1/nodes HTTP endpoints

### ISSUE THREAD

**Issue by p0lyn0mial** [kubermatic/kubermatic#2645]
**Description** `CRUD "/projects/{project_id}/dc/{dc}/clusters/{cluster_id}/nodes"` HTTP endpoints are deprecated and should be removed.

for example this is POST endpoint
https://github.com/kubermatic/kubermatic/blob/f04827f0088d3bcb1ebc44ce02b4abbae0d92aeb/api/pkg/handler/routes_v1.go#L165

**Comment by p0lyn0mial** [comment]
I added `blocked by backed` because the endpoint should be removed after `2.9` release.

**Comment by kdomanski** [comment]
We were thinking about a sensible way to automatically migrate legacy Machines to MachineDeployments, but without breaking customer workloads. Until we come up with a sensible solution, this issue is on hold.

**Comment by kdomanski** [comment]
cc @alvaroaleman @mrIncompetent

**Comment by maciaszczykm** [comment]
It seems like it can be done now. 

@kdomanski Should we delete also an endpoint to list nodes? This endpoint is still used in the UI to list nodes that were created long time ago without deployments.

**Comment by kdomanski** [comment]
These Kubermatic installations should have been updated and their nodes rotated a long time ago. I say burn it.

**Comment by maciaszczykm** [comment]
@zreigz After https://github.com/kubermatic/dashboard/pull/2574 will be merged you can remove all old node endpoints besides the `/delete` endpoint which is still used to delete single nodes from deployment.
